patches/nuget-client/0001-Add-TargetingPack-and-enable-net46-builds-for-source.patch

```diff
@@ -1,28 +1,28 @@
-From a0c69929154c148ccbbf5e833c830d3be512df68 Mon Sep 17 00:00:00 2001
+From 18a85b215bacb7ca8fe0d4cdae3cd591b8899cbc Mon Sep 17 00:00:00 2001
 From: Chris Rummel <crummel@microsoft.com>
 Date: Tue, 12 Jun 2018 18:14:46 -0500
 Subject: [PATCH] Add TargetingPack and enable net46 builds for source-build.
```

Comment:
@crummel is there anything tracking removing this patch I could add a link to?

Reply:
This is https://github.com/NuGet/NuGet.Client/pull/2287.